recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ source:
     - configure.patch
 
 build:
-  number: 0
+  number: 1  # [linux]
+  number: 0  # [win or osx]
```

Comment:
Because nothing has changed for those OSes :-) So I wanted to reflect that we're rebuilding PyQt only on Linux.